CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# CODEOWNERS for autoreview assigning in github
+
+# https://help.github.com/en/articles/about-code-owners#codeowners-syntax
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# Root folder
+*      @krish2718 @jukkar @rado17 @rlubos @sachinthegreen
```

Comment:
Should we adjust these settings ?

Reply:
Sure, although these should be not used by github atm as there is nothing reading the CODEOWNERS file.